src/frontend/Sider/Behandling/Inngangsvilkår/typer/vilkårperiode/målgruppe.ts

```diff
@@ -36,7 +41,7 @@ const målgrupper: Record<Stønadstype, Record<MålgruppeType, boolean>> = {
         OMSTILLINGSSTØNAD: true,
         OVERGANGSSTØNAD: true,
         NEDSATT_ARBEIDSEVNE: true,
-        SYKEPENGER_100_PROSENT: true,
+        SYKEPENGER_100_PROSENT: false,
```

Comment:
Hvordan vil dette påvirke de behandlingene som allerede har `SYKEPENGER_100_PROSENT` ?

Reply:
Testet og gikk fint, får vurdere å legge til noe for å hindre at disse forlenges videre